spec/overcommit/hook_context/pre_push_spec.rb

```diff
@@ -44,6 +44,75 @@
     end
   end
 
+  describe '#modified_files' do
+    let(:remote_url) { "file://#{File.expand_path('some-repo')}" }
```

Comment:
The use of `some-repo` here is cheating a bit as it assumes we know the name of the directory of the repository, which we don't (it is returned by the `repo` helper).

We can work around this by creating the remote repo in the `let` so we have the reference.

```ruby
let(:remote_repo) do
  repo do
    ...
  end
end

let(:remote_url) { "file://#{remote_repo}" }
```

You can then enter the repo in the specs to perform additional custom setup work:

```ruby
it ... do
  Dir.chdir remote_repo do
    ...
  end
end
```

Reply:
Looks good. Will try this out.